apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js

```diff
@@ -136,7 +144,7 @@ const AllResidentBillingReceiptsService = new GQLCustomSchema('AllResidentBillin
                     'OR': receiptsQuery,
                 }
 
-                const receiptsForConsumer = await BillingReceiptAdmin.getAll(
+                const receiptsForConsumer = await ResidentBillingReceiptAdmin.getAll(
```

Comment:
can we use find here?

Reply:
nope, since we have a lot of relations here, that we going to use